ftfy/chardata.py

```diff
@@ -70,11 +67,9 @@ def _build_utf8_punct_regex():
     #
     # Unrelatedly, the expression that generates these bytes will be so much
     # prettier when we deprecate Python 2.
-    continuation_char_list = ''.join(
-        unichr(i) for i in range(0x80, 0xc0)
-    ).encode('latin-1')
+    continuation_bytes = bytes(range(0x80, 0xc0))
```

Comment:
Inline this, perhaps?  (I'm trying to figure out exactly what the obsolete comment above was talking about, and maybe that's the point.)